TESTS/host_tests/pyusb_msd.py

```diff
@@ -0,0 +1,242 @@
+"""
+Copyright (c) 2019, Arm Limited and affiliates.
+SPDX-License-Identifier: Apache-2.0
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+from mbed_host_tests import BaseHostTest
+import time
+import psutil
+import tempfile
+import os
+import platform
+import subprocess
+import sys
+
+
+class PyusbMSDTest(BaseHostTest):
+    """Host side test for USB MSD class."""
+
+    __result = None
+    MOUNT_WAIT_TIME = 25 # in [s]
+    initial_disk_list = None
+    msd_disk = None
+    
+    def _callback_record_disk_list(self, key, value, timestamp):
+        """Records msd disks list.
+
+        It will be used to find new disks mounted during test.
+        """
+        self.initial_disk_list = set(MSDUtils.disks())
```

Comment:
how test would behave if we run multiple ones in same hosts parallel? test looks a bit problematic in ci where >>2 devices are connected in same host and devices appears and disappeared all the time.

Reply:
Currently it will not work.